packages/local_auth/android/src/main/java/io/flutter/plugins/localauth/LocalAuthPlugin.java

```diff
@@ -129,6 +130,12 @@ public void onError(String code, String error) {
         if (Build.VERSION.SDK_INT >= 23) {
           if (packageManager.hasSystemFeature(PackageManager.FEATURE_FINGERPRINT)) {
             biometrics.add("fingerprint");
+          } else {
+            FingerprintManagerCompat fingerprintManager =
```

Comment:
Can you add a comment here why you are adding this including the github issue url? This is *not* the right fix. The right fix is to make sure Xiomi phones support system feature flags. Apparently they don't :(.